src/video/x11/SDL_x11window.c

```diff
@@ -1516,6 +1526,17 @@ static void X11_SetWindowFullscreenViaWM(_THIS, SDL_Window *window, SDL_VideoDis
                            SubstructureNotifyMask | SubstructureRedirectMask, &e);
         }
 
+        if (!fullscreen) {
+            int dest_x = 0, dest_y = 0;
+            SDL_RelativeToGlobalForWindow(window,
+                                          window->windowed.x, window->windowed.y,
+                                          &dest_x, &dest_y);
+
+            /* Attempt to move the window */
+            X11_XMoveWindow(display, data->xwindow, dest_x, dest_y);
```

Comment:
Are we missing the border values here?

Reply:
no, I think this ok!
but there are more stuff to fix I think!

@slouken: just a question to make sure,
SDL_SetWindowPosition(x, y).  
x/y is the top-left of the window decoration border right ? (topleft rendering area, is little after eg (x, y+50))

Reply:
x/y is the top-left of the client rendering area.

Reply:
ok, my mistake, I've reverted that